MuEMU.py

```diff
@@ -0,0 +1,299 @@
+import zipfile
```

Comment:
This might make more sense in .azuredevops/scripts/* but I'm not terribly concerned one way or the other.

Reply:
I think that makes sense.  I moved it.